### PR TITLE
Completed css-three-column-layout challenge - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-30 at 6 56 26 PM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.row {
+  display: flex;
+}
+
+.col-full {
+  text-align: center;
+  color: white;
+}
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: You don't use this class anywhere, so you should delete this.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just one small change requested here Bryan! Very nicely done!

File: css-three-column-layout/styles.css (hunk 1 above)
Comment (bbrown949) [reply css-three-column-layout/styles.css]: done!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**